kaskade/services.py

```diff
@@ -371,11 +410,9 @@ async def _get_watermarks(
     ) -> tuple[int, int]:
         low, high = 0, 0
 
-        consumer = Consumer(self.config | {GROUP_ID: f"kaskade-{uuid.uuid4()}"})
```

Comment:
I remember there was a reason 🤔 , but I need to test it again, then I will document it. But so far looks not necessary